cdap-ui/app/cdap/services/PipelineErrorFactory.js

```diff
@@ -256,8 +260,8 @@ let hasValidConfig = (importConfig) => {
   return importConfig.config;
 };
 let hasValidSchedule = (importConfig) => {
-  let isBatchPipeline = GLOBALS.etlBatchPipelines.indexOf(importConfig.artifact.name) !== -1;
-  return !isBatchPipeline ? true : importConfig.config.schedule;
+  let isSchedulable = GLOBALS.programType[importConfig.artifact.name] === 'workflows';
+  return !isSchedulable ? true : importConfig.config.schedule;
```

Comment:
do `isSchedulable ? importConfig.config.schedule : true` so that it's more readable?

Reply:
Fixed